test/Frontend/skip-function-bodies.swift

```diff
@@ -239,7 +283,8 @@ public struct Struct {
   public var didSetVar: Int = 1 {
     didSet {
       // Body typechecked regardless
-      _blackHole("didSet body") // CHECK-NOT: "didSet body"
+      _blackHole("didSet body") // Do not check for "didSet body" as it's in
+                                // SIL but not the swiftinerface.
```

Comment:
Could add a separate check for the swiftinterface?

Reply:
I just checked this locally - `didSet body` isn't in the serialized SIL for me (CHECK fails, CHECK-NOT succeeds). Am I missing something here?

Also, just noticed typo - "swiftinerface" -> "swiftinterface"

Reply:
Right, I see a few issues if I set it to `CHECK`. First, it fails to find the next string `@_transparent method body`, which is probably an ordering issue, the test succeeds because it had already skipped over that string when looking ahead for the `CHECK-NOT`. And then I see the string in `Skip.noninlinable.sil` but not in `Skip.all.sil`, is that expected? 

Just to be clear on the terminology, I'd call this textual SIL or simply SIL, serialized SIL wouldn't be as readable.

Reply:
Sigh, sorry - I rushed this part of the test. Should have double checked the generated textual SIL. The ordering looks a little weird, the willSet is between publicFunc and privateFunc and the didSet is between the transparent init and inlinable init. Assuming that's stable (?) they should both be moved.

It's definitely expected that it's skipped for -experimental-skip-all-function-bodies - that option just skips outputting SIL entirely. I had thought it would be skipped for non-inlinable as well, but that looks like a misunderstanding on my part.

FWIW, the old "skip-noninline-function-bodies.swift" test had a CHECK prefix for SIL vs interface. I had removed it as it was being used, but this looks like a case for it.